kuksa_databroker/databroker/src/authorization/jwt/decoder.rs

```diff
@@ -171,4 +177,34 @@ AEiqOjPq0D6X45wCzIwjILUCAwEAAQ==
             Err(err) => panic!("decode should succeed but failed with:{}", err),
         }
     }
+
+    #[test]
+    fn test_expiration_safe_addition() {
```

Comment:
Would it possibly be useful with a small comment here specifying the purpose of the test, possibly something like "Rust can only handle date up to XXXX, verify that an error is reported if token expires in year 2424"